packages/adapter-fetch/src/index.js

```diff
@@ -15,6 +15,7 @@ export default next => async ({ endpoint, ...others }) => {
 
   const meta = {};
   resp.headers.forEach((value, key) => (meta[key] = value));
+  meta.responeCode = resp.status;
```

Comment:
I believe there is a typo, ”responseCode” not ”responeCode”.
Can you check if we already had ”statusCode”? Because IMO ”statusCode” is a better name. I know, naming is hard.

Reply:
updated, thanks @khanghoang